Documentation/op-guide/configuration.md

```diff
@@ -312,6 +312,16 @@ The security flags help to [build a secure etcd cluster][security].
 
 ## Logging flags
 
+### --logger
++ Specify 'zap' for structured logging or 'capnslog'.
```

Comment:
Are there any supporting documentation pages describing each output and what they look like?